docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        if axe is None:
</pre><p>Why is this called "axe"? Would rename to "ax", as this is standard in matplotlib naming conventions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>